src/ios/CDVInAppBrowser.m

```diff
@@ -815,6 +827,13 @@ - (void)close
     });
 }
 
+- (void)hide
```

Comment:
As I said before, you can just remove this method if you call browserHide from the another hide